bindings/python/elliptics_session.cpp

```diff
@@ -1531,16 +1549,18 @@ void init_elliptics_session() {
 		     "    new_status = session.remove_backend(elliptics.Address.from_host_port_family(host='host.com', port=1025, family=AF_INET), 0).get()[0].backends[0]")
 
 		.def("start_defrag", &elliptics_session::start_defrag,
-		     (bp::arg("host"), bp::arg("port"), bp::arg("family"), bp::arg("backend_id")),
-		     "start_defrag(host, port, family, backend_id)\n"
+		     (bp::arg("host"), bp::arg("port"), bp::arg("family"), bp::arg("backend_id"),
+		      bp::arg("chunks_dir")=bp::api::object()),
+		     "start_defrag(host, port, family, backend_id, chunks_dir)\n"
 		     "    Start defragmentation of backend @backend_id at node addressed by @host, @port, @family\n"
```

Comment:
Docstring is possibly not enough. New parameter is not described.

Reply:
Updated docstring